components/RuiBreadcrumbs/src/RuiBreadcrumbs.ts

```diff
@@ -0,0 +1,171 @@
+/**
+* Copyright Deloitte Digital 2019
+*
+* This source code is licensed under the BSD-3-Clause license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+import {LitElement, html, property, CSSResultArray, TemplateResult} from 'lit-element';
+import {variables, layout} from './RuiBreadcrumbs.css'
+
+
+export class RuiBreadcrumbs extends LitElement {
+	public constructor() {
+		super();
+		/**
+		 * set child array and collapsed array
+		 * */
+		this._childArray = [...this.children];
+	}
+
+	/**
+	 * Private properties to set number of crumbs before and after collapse element
+	 * */
+	private _itemsBeforeCollapse: number = 1;
+	private _itemsAfterCollapse: number = 1;
+	private _collapsedSlotEl: HTMLSlotElement | null = null;
+
+	private _childArray: any[] = [];
+	private _collapsedArray: any[] = [];
+
+	private _isExpanded: boolean = false;
+	public set expand(val) {
+		const oldVal = this._isExpanded;
+		this._isExpanded = val;
+		this.requestUpdate('expand', oldVal);
+	}
+	public get expand() { return this._isExpanded; }
+
+	/**
+	 * The array of breadcrumbs
+	 */
+	@property({type : String})
+	public separator?: string = '/';
```

Comment:
Separators are purely decoration so shouldn't be rendered as a content.
Please read the 3rd point of the "Accessibility Features" from the WAI-ARIA design pattern: https://www.w3.org/TR/2019/NOTE-wai-aria-practices-1.1-20190207/examples/breadcrumb/index.html